### PR TITLE
added UIAutomator2 rotation test

### PR DESCRIPTION
**Pull request by saikrishna321**
## Change list

Added Rotation tests with UIAutomator2

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

@TikhomirovSergey @SrinivasanTarget review pls

### CODE REVIEW

File: src/test/java/io/appium/java_client/android/UIAutomator2Test.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    /**
+     * initialization.
+     */
+    @BeforeClass public static void beforeClass() throws Exception {
+        service = AppiumDriverLocalService.buildDefaultService();
+        service.start();
+
+        if (service == null || !service.isRunning()) {
+            throw new RuntimeException("An appium server node is not started!");
```
Comment (SrinivasanTarget) [inline src/test/java/io/appium/java_client/android/UIAutomator2Test.java]: @saikrishna321 Should throw AppiumServerHasNotBeenStartedLocallyException.

**Comment by SrinivasanTarget** [comment]
https://github.com/appium/appium/issues/7131

**Comment by SrinivasanTarget** [comment]
https://github.com/appium/java-client/pull/439

**Comment by TikhomirovSergey** [comment]
@saikrishna321 Yes. Could you improve the issue and then it will be merged

**Pull request merged**